tests/e2e/lib.sh

```diff
@@ -618,16 +618,37 @@ collect_and_check_stackrox_logs() {
 # system tests against the same cluster.
 remove_existing_stackrox_resources() {
     info "Will remove any existing stackrox resources"
-
+    local psps_supported=false
```

Comment:
The changes in this file are just reducing some "resource X not found" noise from the logs.

Reply:
Nit: I think that these changes are not needed for the Scanner V4 tests, so I'd extract them into their own PR.

Reply:
Done, will be done separate, thanks.